### PR TITLE
update ancestry manager to use cfg

### PR DESCRIPTION
**Pull request by ScottSuarez**
closes https://github.com/GoogleCloudPlatform/terraform-validator/issues/279

credentials/token were not being used for resourceManager initialization.

### CODE REVIEW

File: ancestrymanager/ancestrymanager_test.go [hunk 1]
```diff
@@ -91,11 +88,8 @@ func TestGetAncestry(t *testing.T) {
 		wantError bool
 		want      string
 	}{
-		{name: "owner_project_online", target: amOnline, query: ownerProject, want: ownerAncestryPath},
```
Comment (melinath) [inline ancestrymanager/ancestrymanager_test.go:94]: Are these tests replaced anywhere / would it be possible to replace them?

**Review by melinath — CHANGES_REQUESTED** [review]
The general approach looks good. I have some small fixes & questions.

File: tfgcv/planned_assets.go [hunk 2]
```diff
@@ -60,15 +59,13 @@ func ReadPlannedAssets(ctx context.Context, path, project, ancestry string, offl
...
-	if err != nil {
-		return nil, errors.Wrap(err, "constructing resource manager client")
-	}
-	converter, err := google.NewConverter(ctx, ancestryManager, project, offline, convertUnchanged)
+	cfg, err := cnvconfig.GetConfig(ctx, project, offline)
+	ua := "config-validator-tf/%s"
+	ancestryManager, err := ancestrymanager.New(cfg, project, ancestry, ua, offline)
 	if err != nil {
-		return nil, errors.Wrap(err, "building google converter")
+		return nil, errors.Wrap(err, "building google configuration")
```
[the remark this answers is not in the document]
Comment (melinath) [reply tfgcv/planned_assets.go:66]: (Currently errors from GetConfig are effectively ignored.)

File: ancestrymanager/ancestrymanager_test.go [hunk 3]
```diff
@@ -75,11 +75,8 @@ func TestGetAncestry(t *testing.T) {
 	ts := newAncestryManagerMockServer(t, cache)
 	defer ts.Close()
 
-	amOnline, err := New(ctx, ownerProject, "", false, option.WithEndpoint(ts.URL), option.WithoutAuthentication())
-	if err != nil {
-		t.Fatalf("failed to create online ancestry manager: %s", err)
-	}
-	amOffline, err := New(ctx, ownerProject, ownerAncestry, true)
+	cfgOffline, err := cnvconfig.GetConfig(ctx, anotherProject, false)
```
[the remark this answers is not in the document]
Comment (ScottSuarez) [reply ancestrymanager/ancestrymanager_test.go:78]: oof, nice. moved things around too much

File: converters/google/convert_test.go [hunk 4]
```diff
@@ -33,91 +32,21 @@ const testProject = "test-project"
...
+	cfg, err := cnvconfig.GetConfig(ctx, project, offline)
 	if err != nil {
-		return nil, errors.Wrap(err, "constructing resource manager client")
+		return nil, errors.Wrap(err, "constructing configuration")
 	}
-	c, err := NewConverter(ctx, ancestryManager, project, offline, convertUnchanged)
+	ancestryManager, err := ancestrymanager.New(cfg, project, ancestry, ua, offline)
 	if err != nil {
-		return nil, errors.Wrap(err, "building converter")
+		return nil, errors.Wrap(err, "constructing resource manager client")
```
[the remark this answers is not in the document]
Comment (ScottSuarez) [reply converters/google/convert_test.go:44]: done done done

**Review by melinath — APPROVED** [review]
LGTM

**Pull request merged**